ceph/ceph_cluster.yml

```diff
@@ -6,25 +6,29 @@ parameters:
  prefix: ceph
  numcpus: 2
  memory: 4096
- image: centos8
+ image: fedora32
  notify: false
  admin_password: password
  disks:
  - 150
- - 150
+ - 15
```

Comment:
What about something less that this? 150 x 3 means (potentially) 450 GB. I tried with 10 GB it works ok for the OS disk, and for the OSD disks something less than that (5 GB?).

Reply:
I reduced it to 15 for OS and 5 for OSDs.. less space starts to be dangerous if you want to use the cluster for "long" tests using data storage.